### PR TITLE
fix: [CV-8847]: SLO target and error budget reset changes

### PR DESCRIPTION
**Pull request by shashidhargg**
##### Summary:

- SLO target in decimal is showing browser help to set it in a natural number
- Error budget reset UI changes

##### Jira Links:

https://harness.atlassian.net/browse/CVS-8847

##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by shashidhargg** [comment]
retrigger titlecheck

**Review by deepesh-ui — APPROVED** [review]

File: src/modules/85-cv/hooks/useErrorBudgetRestHook/useErrorBudgetRestHook.utils.ts [hunk 1]
```diff
@@ -5,22 +5,30 @@
  * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
  */
 
-export const calculateErrorBudgetByIncrement = (currentErrorBudget: number, increment = NaN): number => {
+export const calculateErrorBudgetByIncrement = (currentErrorBudget: number, increment = NaN): string => {
   if (Number.isNaN(increment)) {
-    return 0
+    return '--'
```
Comment (praneshg239) [inline src/modules/85-cv/hooks/useErrorBudgetRestHook/useErrorBudgetRestHook.utils.ts:10]: shall we move this to `.yaml` or constants?

File: src/modules/85-cv/hooks/useErrorBudgetRestHook/useErrorBudgetRestHook.utils.ts [hunk 2]
```diff
@@ -5,22 +5,30 @@
...
   }
 
-  return Math.floor(currentErrorBudget + currentErrorBudget * (increment / 100))
+  return Math.floor(currentErrorBudget + currentErrorBudget * (increment / 100)).toLocaleString()
 }
 
 export const calculateRemainingErrorBudgetByIncrement = (
```
Comment (praneshg239) [inline src/modules/85-cv/hooks/useErrorBudgetRestHook/useErrorBudgetRestHook.utils.ts:16]: it looks like complex calculation, why are doing this in frontend?

File: src/modules/85-cv/hooks/useErrorBudgetRestHook/useErrorBudgetRestHook.utils.ts (hunk 1 above)
Comment (shashidhargg) [reply src/modules/85-cv/hooks/useErrorBudgetRestHook/useErrorBudgetRestHook.utils.ts:10]: Not necessary. It's doesn't do anything with language change.

File: src/modules/85-cv/hooks/useErrorBudgetRestHook/useErrorBudgetRestHook.utils.ts (hunk 2 above)
Comment (shashidhargg) [reply src/modules/85-cv/hooks/useErrorBudgetRestHook/useErrorBudgetRestHook.utils.ts:16]: No, there is nothing complex. We are just doing an increment on the error budget. And it should be done in FE only as users can increase the error budget % by %.

Comment (praneshg239) [reply src/modules/85-cv/hooks/useErrorBudgetRestHook/useErrorBudgetRestHook.utils.ts:16]: got it

**Review by praneshg239 — APPROVED** [review]

**Pull request merged**